### ISSUE TITLE
`clojure.core/test` cannot be resolved

### ISSUE THREAD

**Issue by MrEbbinghaus** [babashka/sci#640]
**version** 0.2.7
**platform** JVM
**problem** [`clojure.core/test`](https://clojuredocs.org/clojure.core/test) is missing from [`sci.impl.namespaces/clojure-core`](https://github.com/babashka/sci/blob/20616983a8be69f2c28c4ef453e7a4eb644a12ee/src/sci/impl/namespaces.cljc#L765)

I can't see any reason why sci shouldn't support `test`. Probably it was just forgotten?

**repro**
```clojure
(defn square 
  {:test #(assert (= 4 (square 2)))} 
  [x] 
  (* x x))
  
(test #'square)

; =>
#error{:cause "Could not resolve symbol: test",
       :data {:type :sci/error, :line 1, :column 2, :file nil, :phase "analysis"},
       :via [{:type clojure.lang.ExceptionInfo,
              :message "Could not resolve symbol: test",
              :data {:type :sci/error, :line 1, :column 2, :file nil, :phase "analysis"},
              :at [sci.impl.utils$throw_error_with_location invokeStatic "utils.cljc" 21]}]}
```

**expected behavior**
```clojure

(test #'square)
; => :ok

(defn broken-square 
  {:test #(assert (= 4 (broken-square 2)))} 
  [x] 
  (+ x x))
  
(test #'broken-square )
; =>
; Execution error (AssertionError) at user/fn (REPL:1).
; Assert failed: (= 5 (broken-square 2))
```

**Comment by borkdude** [comment]
Both babashka and nbb have a clojure test implementation for SCI, but it isn't available (yet) in SCI itself. I could port those configurations back as add-ons though.

**Comment by borkdude** [comment]
oh wait, `clojure.core/test`: that one is much easier!

**Comment by borkdude** [comment]
Fixed with 53955c37cecc79250ebe2de4c435ca57572135f9
